picklayer/__init__.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
@jampukka could you do some review? Especially could you check the copyright notices?